package.json

```diff
@@ -18,56 +32,25 @@
     "loaders.css": "^0.1.2",
     "lodash": "^4.17.15",
     "node-fetch": "^2.6.0",
-    "prop-types": "15.7.2",
     "react": "16.12.0",
     "react-dom": "16.12.0",
     "react-helmet": "5.2.1",
     "react-icons": "^3.8.0"
   },
-  "keywords": [
-    "gatsby",
-    "tailwind",
-    "tailwindcss",
-    "purgecss"
-  ],
-  "license": "MIT",
-  "scripts": {
-    "analyze:lint": "eslint --ext .jsx --ext .js .",
-    "analyze:prettier": "prettier --list-different '**/*.{js,css,md,json}'",
-    "analyze": "npm run analyze:lint && npm run analyze:prettier",
-    "build": "gatsby build",
-    "develop": "gatsby develop",
-    "fix:lint": "eslint --ext .jsx --ext .js . --fix",
-    "fix:prettier": "prettier --write '**/*.{js,css,md,json}'",
```

Comment:
Why did you decide to remove `fix` command?

Reply:
I left only `lint` command with `--fix`

Reply:
Those should be two different commands. One is executed to fix stuff, while other is executed in CI to make sure code is formatted correctly